cmd/lint.go

```diff
@@ -46,10 +46,10 @@ func lintCommandAction(cmd *cobra.Command, args []string) error {
 
 	ok, err := docs.IsReadmeUpToDate()
 	if err != nil {
-		return errors.Wrapf(err, "can't check if %s file is up-to-date", docs.ReadmeFile)
+		return errors.Wrap(err, "can't check if file is up-to-date")
 	}
 	if !ok {
-		return fmt.Errorf("%s file is outdated. Rebuild the package with 'elastic-package build'", docs.ReadmeFile)
+		return fmt.Errorf("readme file is outdated. Rebuild the package with 'elastic-package build'")
```

Comment:
Yeah, I know this way is simpler :) but I believe it's still valid to show which files need to be rebuild. I know that the build step renders all of them at once, but it would consistent with general rules in elastic-package - to report exact files that are causing issues.

Reply:
++ to @mtojek's suggestion. We should strive to make error messages emitted by `elastic-package` easily actionable for the user.

Reply:
Agree! Done!!